rados/snapshot_test.go

```diff
@@ -0,0 +1,61 @@
+package rados
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func (suite *RadosTestSuite) TestCreateRemoveSnapshot() {
+	suite.SetupConnection()
+
+	// Happy path.
+	suite.T().Run("NewSnap", func(t *testing.T) {
+		ta := assert.New(t)
+		snapName := "mySnap"
```

Comment:
can you create a snapshot with an empty name?

Reply:
Yes, I have a test case for that where I pass empty string and CreateSnapshot returns no error.